vcpkg.json

```diff
@@ -1,37 +1,41 @@
 {
+  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
   "name": "communityshaders",
-  "version-semver": "0.7.4",
```

Comment:
Why are we removing the version?

Reply:
It's only meant to be used for libraries